db/schema.rb

```diff
@@ -210,6 +210,17 @@
     t.index ["name"], name: "index_disabilities_on_name", unique: true
   end
 
+  create_table "dqt_trn_requests", force: :cascade do |t|
```

Comment:
Need to add this table and fields to analytics.yml. Probably without the response but we should check that with @AndrewPetrie1

Reply:
We might want to add the response to sanitise.yml too if we're going to consider it PII.